scripts/docker_start_user.sh

```diff
@@ -90,7 +90,7 @@ function setup_apollo_directories() {
 # FIXME(infra): This will change core pattern on the host also,
 # where the `/apollo` directory may not exist.
 function setup_core_pattern() {
-  if [ -e /proc/sys/kernel ]; then
+  if [ -e /proc/sys/kernel -a -w /proc/sys/kernel/core_pattern ]; then
     echo "/apollo/data/core/core_%e.%p" > /proc/sys/kernel/core_pattern
   fi
 }
```

Comment:
Shellcheck suggests:

```
In hello.sh line 3:
if [ -e /proc/sys/kernel -a -w /proc/sys/kernel/core_pattern ]; then
                         ^-- SC2166: Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.
```

Reply:
Interesting, I'm using -a all the time inside single braces as it's documented in test https://linux.die.net/man/1/test.

Will update it to:
```
if [[ -w /proc/sys/kernel/core_pattern ]]; then
```

to make it even simpler.